.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Would you please remove the .idea directory from the PR?
Feel free to add `/.idea` to the root `.gitignore`.